src/Satooshi/Component/System/Git/GitCommand.php

```diff
@@ -39,9 +39,11 @@ public function getBranches()
      */
     public function getHeadCommit()
     {
-        $command = $this->createCommand("log -1 --pretty=format:'%H\n%aN\n%ae\n%cN\n%ce\n%s'");
+        $command = $this->createCommand("log -1 --pretty=format:'%H:%aN:%ae:%cN:%ce:%s'");
```

Comment:
That is why I limit the explode. O will try your suggestion on Monday
2 sty 2016 18:02 "Takashi Matsuo" <notifications@github.com> napisał(a):

> In src/Satooshi/Component/System/Git/GitCommand.php
> <https://github.com/satooshi/php-coveralls/pull/176#discussion_r48685824>:
>
> > @@ -39,9 +39,11 @@ public function getBranches()
> >       */
> >      public function getHeadCommit()
> >      {
> > -        $command = $this->createCommand("log -1 --pretty=format:'%H\n%aN\n%ae\n%cN\n%ce\n%s'");
> > +        $command = $this->createCommand("log -1 --pretty=format:'%H:%aN:%ae:%cN:%ce:%s'");
>
> According to https://git-scm.com/docs/pretty-formats
> I think you can use '%n' for newline (I also actually confirmed it's
> working on my windows machine).
>
> A concern with using ':' is that other parts may contain ':' and it will
> break something.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/satooshi/php-coveralls/pull/176/files#r48685824>.
>